jupyterhub/app.py

```diff
@@ -356,6 +356,27 @@ def _validate_config_file(self, proposal):
         Default is two weeks.
         """,
     ).tag(config=True)
+
+    oauth_token_expires_in = Integer(
+        24 * 3600,
+        help="""Expiry (in seconds) of OAuth access tokens.
+
+        These are the tokens stored in cookies when you visit
+        a single-user server or service.
+        When they expire, you must re-authenticate with the Hub,
```

Comment:
Can you add a comment here perhaps about API tokens that never need refreshing? Or should that be elsewhere?

Reply:
API tokens *may* expire, depending on how they are issued, but the default is no expiry.

Reply:
Added a note that this doesn't apply to API tokens